app/views/includes/head.njk

```diff
@@ -14,7 +14,13 @@
   ga('set', 'displayFeaturesTask', null);
   ga('set', 'transport', 'beacon');
 
-  ga('send', 'pageview');
+  var url = new URL(document.location.href);
+  var search_params = new URLSearchParams(url.search);
+  search_params.delete('email');
+  url.search = search_params.toString();
+  var filtered_page_path = url.pathname + url.search;
```

Comment:
didn't want to spend too much time on review, but **probably** there is a nicer way of doing it.

Reply:
probably. but i'm no javascript developer. if it works, then i'd rather do this to get something out so we can meet our GDPR legal requirements asap.